GSoCResources/UmbrellaOrganisations.md

```diff
@@ -1,9 +1,99 @@
-# What are Umbrella Organisations? 
+# Umbrella Organizations in Google Summer of Code
 
-This is a term which you may or may not have heard of in the Open Source world as such organisations are more prevelant or rather relevant inside the Google Summer of Code framework. Umbrella organisations are basically large oorganisations which have smaller sub-organisations under it. 
+## What are Umbrella Organizations? 
 
-The reason for their clustering under a parent organisation varies. There might be several factors for such a structure.
-* These organisations are funded/supported by the parent organisation or are branches of the organisation.
-* Small/first-time organisations come together as an organisation to increase their chances of selection into their programme. 
-* Small/first-time organisation merges under a well established organisation to increase their chances of selection the programme. 
-* A strong language based organisation which floats several of it's popular modules as its suborganisations.
+This is a term which you may or may not have heard of in the Open Source world as such organizations are more prevalent or rather relevant inside the Google Summer of Code (GSoC) framework. Umbrella organizations are basically large organizations which have smaller sub-organizations under it. 
+
+The reason for their clustering under a parent organization varies. There might be several factors for such a structure.
+* These organizations are funded/supported by the parent organization or are branches of the organization.
+* Small/first-time organizations come together as an organization to increase their chances of selection into their program. 
+* Small/first-time organization merges under a well established organization to increase their chances of selection the program. 
+* A strong language based organization which floats several of it's popular modules as its sub-organizations.
+
+## List of Umbrella Organizations
+
+You can find the list of Umbrella Organizations along with the sub-organizations that participate in GSoC. Please note that this list corresponds to the organizations that participated in GSoC 2017. The organizations may or may not decide to participate in the future.
+
+1. **BRL-CAD**
+
+    - BRL-CAD - 3D solid modeling, ray tracing, and geometric analysis.
```

Comment:
Do we want to add links? _That_ would be immensely cumbersome; something I am not ready to do by myself. But of course, people are welcome to contribute.